src/main/java/duke/Deadlines.java

```diff
@@ -0,0 +1,36 @@
+package duke;

```

Comment:
Maybe can include package documentation also

Reply:
Okay! Thank you!